mix.exs

```diff
@@ -36,14 +36,12 @@ defmodule GRPC.Mixfile do
 
   defp deps do
     [
-      {:protobuf, "~> 0.5"},
-      {:cowboy, "~> 2.7"},
-      {:gun, "~> 2.0.0", hex: :grpc_gun},
-      # 2.9.0 fixes some important bugs, so it's better to use ~> 2.9.0
-      # {:cowlib, "~> 2.9.0", override: true},
-      {:ex_doc, "~> 0.23", only: :dev},
+      {:protobuf, "~> 0.8.0-beta.1"},
+      {:cowboy, "~> 2.9"},
+      {:gun, "~> 2.0.0-rc.2"},
```

Comment:
no changes compare to official version https://github.com/ninenines/gun/compare/master...elixir-grpc:v2.0